src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.Forwards.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
```

Comment:
most projects have just been putting these in the main .cs file instead of a separate .Forwards file.